src/Game.cpp

```diff
@@ -67,7 +67,7 @@
 using namespace openblack::lhscriptx;
 using namespace std::chrono_literals;
 
-const std::string k_WindowTitle = "openblack";
+const std::string k_windowTitle = "openblack";
```

Comment:
warning: invalid case style for global constant 'k_windowTitle' [readability-identifier-naming]

```suggestion
const std::string k_WindowTitle = "openblack";
```

src/Game.cpp:117:
```diff
- 		    std::make_unique<GameWindow>(k_windowTitle, args.windowWidth, args.windowHeight, args.displayMode, extraFlags);
+ 		    std::make_unique<GameWindow>(k_WindowTitle, args.windowWidth, args.windowHeight, args.displayMode, extraFlags);
```